# Computational genomics papers

## Quality control and batch effects

 - AC't Hoen, Peter, et al. "[Reproducibility of high-throughput mRNA and small RNA sequencing across laboratories](http://dx.doi.org/10.1038/nbt.2702)." _Nature biotechnology_ 31.11 (2013): 1015.
 - Leek, Jeffrey T., et al. "[Tackling the widespread and critical impact of batch effects in high-throughput data.](http://dx.doi.org/10.1038/nrg2825)" _Nature Reviews Genetics_ 11.10 (2010): 733.
 - Panousis, Nikolaos I., et al. "[Allelic mapping bias in RNA-sequencing is not a major confounder in eQTL studies.](https://doi.org/10.1186/s13059-014-0467-2)" _Genome biology_ 15.9 (2014): 467.
 - Castel, Stephane E., et al. "[Tools and best practices for data processing in allelic expression analysis.](http://dx.doi.org/10.1186/s13059-015-0762-6)" _Genome biology_16.1 (2015): 195.

## Regulatory genetic variation

 - Albert, Frank W., and Leonid Kruglyak. "[The role of regulatory variation in complex traits and disease.](http://dx.doi.org/10.1038/nrg3891)" _Nature Reviews Genetics_ 16.4 (2015): 197. 

## General advice

 - Mensh, Brett, and Konrad Kording. "[Ten simple rules for structuring papers](https://doi.org/10.1371/journal.pcbi.1005830)." _PLoS computational biology_ 13.9 (2017): e1005619.
 - Wilson, Greg, et al. "[Good enough practices in scientific computing.](http://dx.doi.org/10.1371/journal.pcbi.1005510)" _PLoS computational biology_ 13.6 (2017): e1005510.
 - Wilson, Greg, et al. "[Best practices for scientific computing.](http://dx.doi.org/10.1371/journal.pbio.1001745)" _PLoS biology_ 12.1 (2014): e1001745.

<!--stackedit_data:
eyJoaXN0b3J5IjpbMTIzMjc4MDM2OSwtMTkzMjI0MTIxNCw4Mz
k3ODQ3ODAsLTIwNTg1ODE2MjcsMTcxNzEwNTk2Nyw1NDQ1Nzgw
MTQsMTA3NzA1MDgxNywtMTQxMDE2MDU0OSwtMTUwMjEyMzUyNl
19
-->